api/graphql/index.js

```diff
@@ -1,34 +1,43 @@
-import { buildSchema } from 'graphql'
+// import { buildSchema } from 'graphql'
 import { readFileSync } from 'fs'
 import graphqlHTTP from 'express-graphql'
 import { join } from 'path'
 import Fetcher from './fetcher'
 import models from './models'
+import { makeExecutableSchema } from 'graphql-tools'
 
 const schemaText = readFileSync(join(__dirname, 'schema.graphql')).toString()
-const schema = buildSchema(schemaText)
-
-const createRootValue = (userId, isAdmin) => {
-  if (!userId) {
-    return {
-      me: null
-    }
-  }
+// const schema = buildSchema(schemaText)
 
+function createSchema (userId, isAdmin) {
   const fetcher = new Fetcher(models(userId, isAdmin))
 
-  return {
-    me: () => fetcher.fetchOne('me', userId),
-    community: ({ id, slug }) => // you can specify id or slug, but not both
-      fetcher.fetchOne('communities', slug || id, slug ? 'slug' : 'id'),
-    person: ({ id }) => fetcher.fetchOne('users', id),
+  const resolvers = Object.assign({
+    Query: {
+      me: () => fetcher.fetchOne('me', userId),
+      community: (root, { id, slug }) => // you can specify id or slug, but not both
+        fetcher.fetchOne('communities', slug || id, slug ? 'slug' : 'id'),
+      person: (root, { id }) => fetcher.fetchOne('users', id)
+    },
+    Mutation: {
+      updateMe: (root, { changes }) => {
+        return User.find(userId)
+        .then(user => user.validateAndSave(changes))
+        .then(() => fetcher.fetchOne('me', userId))
+      }
+    },
 
-    updateMe: ({ changes }) => {
-      return User.find(userId)
-      .then(user => user.validateAndSave(changes))
-      .then(() => fetcher.fetchOne('me', userId))
+    FeedItemContent: {
```

Comment:
TBD whether we just add these in here manually, or if they should go in `Fetcher` somehow